lib_term/analysis.ml

```diff
@@ -67,7 +67,7 @@ module Make (Job : sig type id end) = struct
     make ~env (Map_input {source; info})
       (match info with
        | Ok _ -> Pass
-       | Error `Blocked -> Blocked)
+       | Error (`Blocked | `Empty_list) -> Blocked)
```

Comment:
We should be careful that this doesn't produce ambiguous DAGs

Reply:
It displays `(empty list)` on the input too, which should make it clear.